src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -109,59 +128,66 @@ function ChplCronGen(props) {
-          <div className={classes.time}>
-            <ChplTextField
-              id="run-time"
-              name="runTime"
-              label="Run Time"
-              type="time"
-              required
-              value={formik.values.runTime}
-              onChange={formik.handleChange}
-              onBlur={formik.handleBlur}
-              error={formik.touched.runTime && !!formik.errors.runTime}
-              helperText={formik.touched.runTime && formik.errors.runTime}
-            />
+          <div className={classes.subContent}>
+            <div>
+              <Typography variant="subtitle2">Every:</Typography>
+              <div className={classes.day}>
+                <FormControlLabel
+                  label="Sunday"
+                  control={<Checkbox name="days" value="SUN" onChange={handleDays} checked={days.has('SUN')} />}
+                />
+                <FormControlLabel
+                  label="Monday"
+                  control={<Checkbox name="days" value="MON" onChange={handleDays} checked={days.has('MON')} />}
+                />
+                <FormControlLabel
+                  label="Tuesday"
+                  control={<Checkbox name="days" value="TUE" onChange={handleDays} checked={days.has('TUE')} />}
+                />
+                <FormControlLabel
+                  label="Wednesday"
+                  control={<Checkbox name="days" value="WED" onChange={handleDays} checked={days.has('WED')} />}
+                />
+                <FormControlLabel
+                  label="Thursday"
+                  control={<Checkbox name="days" value="THU" onChange={handleDays} checked={days.has('THU')} />}
+                />
+                <FormControlLabel
+                  label="Friday"
+                  control={<Checkbox name="days" value="FRI" onChange={handleDays} checked={days.has('FRI')} />}
+                />
+                <FormControlLabel
+                  label="Saturday"
+                  control={<Checkbox name="days" value="SAT" onChange={handleDays} checked={days.has('SAT')} />}
+                />
+              </div>
+            </div>
+            <div>
+              <Typography gutterBottom variant="subtitle2">At:</Typography>
+              <div className={classes.time}>
+                <ChplTextField
+                  id="run-time"
+                  name="runTime"
+                  label="Run Time"
+                  type="time"
+                  required
+                  value={formik.values.runTime}
+                  onChange={formik.handleChange}
+                  onBlur={formik.handleBlur}
+                  error={formik.touched.runTime && !!formik.errors.runTime}
+                  helperText={formik.touched.runTime && formik.errors.runTime}
+                />
+              </div>
+              <FormHelperText className={classes.helperTextSpacing} id="EST-helper-text">All times should be entered as Eastern Time (ET)</FormHelperText>
```

Comment:
What's the `id` for? If it's required that's fine, but if so, it should follow the standard case rules, which for id fields are "snake-case", meaning all lowercase, and with the words separated by dashes